apps/condo/domains/common/constants/style.js

```diff
@@ -29,6 +29,12 @@ const lightRed = '#FFF1F0'
 const brightRed = '#F5222D'
 const whiteTranslucent = 'rgba(255, 255, 255, 0.9)'
 
+const CHART_COLOR_SET = [blue[5], green[5], red[4], gold[5], volcano[5], purple[5],
```

Comment:
it's better to move it mear usage. It's useless as global domain const

Reply:
@pahaz this color scheme will be used in the future for another analytics reports